### PR TITLE
fix issue with node label click opening overview

### PR DESCRIPTION
**Pull request by invincibleJai**
fix the issue with node label click

Addresses issue as per comment in https://github.com/openshift/console/pull/2335#pullrequestreview-274347249

GIF: 

ezgif com-video-to-gif (1)

### CODE REVIEW

File: frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss [hunk 1]
```diff
@@ -9,6 +9,6 @@
 .odc-resource-icon-deployment,
 .odc-resource-icon-daemonset,
 .odc-resource-icon-deploymentconfig {
-  cursor: pointer;
+  cursor: auto;
```
Comment (vikram-raj) [inline frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss]: No need to add cursor property. Default is fine.

Comment (invincibleJai) [reply frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss]: With Default, we won't get the hover effect as we have for other resources badges. So added.

Comment (vikram-raj) [reply frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss]: Got it. Thanks.

**Pull request merged**